Mathlib/Algebra/Ring/CentroidHom.lean

```diff
@@ -101,9 +101,9 @@ instance : CentroidHomClass (CentroidHom α) α where
   map_mul_right f := f.map_mul_right'
 
 
-/-- Helper instance for when there's too many metavariables to apply `FunLike.CoeFun`
+/-- Helper instance for when there's too many metavariables to apply `DFunLike.CoeFun`
 directly. -/
-/- Porting note: Lean gave me `unknown constant 'FunLike.CoeFun'` and says `CoeFun` is a type
+/- Porting note: Lean gave me `unknown constant 'DFunLike.CoeFun'` and says `CoeFun` is a type
```

Comment:
Do you think editing this porting note is OK?

Reply:
Not sure: it would be false as written so probably revert this? Although I suppose the error would involve `DFunLike` if we ported now, so maybe it's okay. Probably we can delete this instance anyway (after #8386). 

```suggestion
/- Porting note: Lean gave me `unknown constant 'FunLike.CoeFun'` and says `CoeFun` is a type
```